components/pipedrive/actions/update-deal/update-deal.mjs

```diff
@@ -1,116 +1,120 @@
-// legacy_hash_id: a_vgi48J
-import { axios } from "@pipedream/platform";
+import pipedriveApp from "../../pipedrive.app.mjs";
 
 export default {
   key: "pipedrive-update-deal",
   name: "Update Deal",
-  description: "Updates the properties of a deal.",
-  version: "0.1.1",
+  description: "Updates the properties of a deal. See the Pipedrive API docs for Deals [here](https://developers.pipedrive.com/docs/api/v1/#!/Deals)",
```

Comment:
I think this would be a better doc link:
```suggestion
  description: "Updates the properties of a deal. See the Pipedrive API docs for Deals [here](https://developers.pipedrive.com/docs/api/v1/Deals#updateDeal)",
```